# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: ipc/ipdl/message-metadata.ini
# Commit: e6c12eec1cef
# Full Hash: e6c12eec1cef32987b7a0b79ccf4420ee097b868
# Author: Junior Hsu <juhsu@mozilla.com>
# Date: 2020-06-30 10:00:42
# Regressor Bug: 1633935
# File Overlap Count: 1
# Description:
#   Bug 1633935 - P4 Remove On[Start|Stop]Request/OnTransportAndData in PHttpChannel, r=mayhemer,nika,necko-reviewers
#   
#   We move OnStartRequest from PHttpChannel to PHttpBackgroundChannel, thus adjusting
#   message-metadata.ini
#   
# ==============================================================================

diff -r d59b42f58f86 -r e6c12eec1cef ipc/ipdl/message-metadata.ini
--- a/ipc/ipdl/message-metadata.ini	Tue Jun 30 00:32:53 2020 +0000
+++ b/ipc/ipdl/message-metadata.ini	Tue Jun 30 00:33:01 2020 +0000
@@ -12,9 +12,9 @@
 segment_capacity = 8192
 [PContent::StoreAndBroadcastBlobURLRegistration]
 segment_capacity = 8192
-[PHttpChannel::OnStartRequest]
+[PHttpChannel::Redirect1Begin]
 segment_capacity = 8192
-[PHttpChannel::Redirect1Begin]
+[PHttpBackgroundChannel::OnStartRequest]
 segment_capacity = 8192
 [PHttpBackgroundChannel::OnTransportAndData]
 segment_capacity = 8192